Core/include/Acts/Seeding/SeedFilter.hpp

```diff
@@ -40,27 +39,50 @@ class SeedFilter {
   /// @param invHelixDiameterVec vector containing 1/(2*r) values where r is the helix radius
   /// @param impactParametersVec vector containing the impact parameters
   /// @param zOrigin on the z axis as defined by bottom and middle space point
+  /// @param numQualitySeeds number of high quality seeds in seed confirmation
+  /// @param numSeeds number of low quality seeds in seed confirmation
```

Comment:
ping

Reply:
If `seedConfirmation = false` this is the total number of seeds, but if `seedConfirmation = true` this will be the others seeds that are not "high quality" which I call lower quality. I will explain that in the comment